pkg/globalnet/controllers/controllers_suite_test.go

```diff
@@ -506,6 +505,7 @@ func awaitStatusConditions(client dynamic.ResourceInterface, name string, atInde
 	var conditions []metav1.Condition
 	var notFound error
 
+	//nolint:staticcheck // Replacement unclear, see https://github.com/kubernetes/kubernetes/issues/116712
```

Comment:
I encountered this too.  The replacement is [PollUntilContextTimeout](https://github.com/submariner-io/lighthouse/blob/65fa08c9fd02335a44b3e9ee4b02f5c682ee9e2a/pkg/agent/controller/controller_suite_test.go#L562).